storage/iam.js

```diff
@@ -0,0 +1,190 @@
+  // const roleName = "roles/storage.objectViewer";
+
+  // The list of IAM members to grant the role to
+  // const members = ['user:jdoe@example.com', 'group:admins@example.com'];
+
+  // Imports the Google Cloud client library
+  const Storage = require('@google-cloud/storage');
+
+  // Instantiates a client
+  const storage = Storage();
+
+  // Get a reference to a Google Cloud Storage bucket
+  const bucket = storage.bucket(bucketName);
+
+  // Gets and updates the bucket's IAM policy
+  bucket.iam.getPolicy()
+    .then((data) => {
+      const policy = data[0];
+
+      // Finds and updates the appropriate role-member group
+      const index = policy.bindings.findIndex((role) => role.role === roleName);
+      let role = policy.bindings[index];
+      if (role) {
+        role.members = role.members.filter((member) => members.indexOf(member) === -1);
+
+        // Updates the policy object with the new (or empty) role-member group
+        if (role.members.length === 0) {
+          policy.bindings.splice(index, 1);
+        } else {
+          policy.bindings.index = role;
+        }
+
+        // Updates the bucket's IAM policy
+        return bucket.iam.setPolicy(policy);
+      } else {
+        // No matching role-member group(s) were found
+        throw new Error('No matching role-member group(s) found.');
+      }
+    })
+    .then(() => {
+      console.log(`Removed the following member(s) with role ${roleName} from ${bucketName}:`);
+      members.forEach((member) => {
+        console.log(`  ${member}`);
+      });
+    })
+    .catch((err) => {
+      console.error('Error:', err);
+    });
+  // [END remove_bucket_iam_member]
+}
+
+const cli = require(`yargs`)
+  .demand(1)
+  .array('members')
+  .command(
+    `view-members <bucketName>`,
+    `Lists IAM member-role groups for a given Google Cloud Storage bucket.`,
+    {},
+    (opts) => viewBucketIamMembers(opts.bucketName)
+  )
+  .command(
+    `add-members <bucketName> <roleName> [members..]`,
+    `Adds one or more IAM member-role groups to a Google Cloud Storage bucket.`,
+    {},
+    (opts) => addBucketIamMember(opts.bucketName, opts.roleName, opts.members)
+  )
+  .command(
+    `remove-members <bucketName> <roleName> [members..]`,
+    `Removes one or more IAM member-role groups from a Google Cloud Storage bucket.`,
+    {},
+    (opts) => removeBucketIamMember(opts.bucketName, opts.roleName, opts.members)
+  )
+  .example(`node $0 view-members "my-bucket"`)
+  .example(`node $0 add-members "my-bucket" "storage.objectViewer" "user:jdoe@example.com" "group:admins@example.com"`)
+  .example(`node $0 remove-members "my-bucket" "storage.objectViewer" "user:jdoe@example.com" "group:admins@example.com"`)
+  .wrap(120)
+  .recommendCommands()
+  .epilogue(`For more information, see https://cloud.google.com/iam/docs/overview and https://cloud.google.com/storage/docs`)
+  .help();
+
+if (module === require.main) {
+  cli.help().strict().argv; // eslint-disable-line
```

Comment:
Maybe you're right, perhaps we should make this the standard (a "main" function).

Change to:

```js
  .epilogue(`For more information, see https://cloud.google.com/iam/docs/overview and https://cloud.google.com/storage/docs`)
  .help()
  .strict();

if (module === require.main) {
  cli.parse(process.argv.slice(2));
}
```